Add int maintenance alert to all pages

orenyk opened the pull request:
This can be turned on by specifying the timeframe in _config.yml and
turned off by setting the `int_maintenance_window` configuration value
to `false`.

NatashaJIbrahim reviewed (approved):
<3

NatashaJIbrahim reviewed (commented):
Just thought of one slight improvement ... just to add this to the README. I can do that if you don't get to it.

orenyk commented:
@NatashaJIbrahim I refactored to make it more general (you can set any arbitrary site-wide alert) and added info to the README. Will merge once the build passes!
